ui/tests/helpers/secret-edit-toolbar.js

```diff
@@ -0,0 +1,12 @@
+import { click } from '@ember/test-helpers';
+const SELECTORS = {
+  dropdown: '[data-test-copy-menu-trigger]',
+  wrapButton: '[data-test-wrap-button]',
+};
+export default async function assertSecretWrap(assert, context, path) {
```

Comment:
since we're passing `assert` separately instead of accessing on `context` could we simplify by passing `server` instead on `context`? 
```suggestion
export default async function assertSecretWrap(assert, server, path) {
```

Reply:
Yeah!